book/01-introduction/sections/about-version-control.asc

```diff
@@ -1,23 +1,23 @@
-=== About Version Control
+=== Versiyon Kontrol Sistemleri
 
 (((version control)))
-What is ``version control'', and why should you care?
-Version control is a system that records changes to a file or set of files over time so that you can recall specific versions later.
-For the examples in this book, you will use software source code as the files being version controlled, though in reality you can do this with nearly any type of file on a computer.
+"Versiyon kontrol" nedir ve neden önemsenmelidir?
+Versiyon kontrol sistemi; dosya ve dosya grupları üzerinde yapılan değişiklikleri versiyonlar şeklinde kayıt edip istenilen zamanda ilgili versiyonlara erişilebilmesini sağlayan sistemdir.
```

Comment:
Soylediginiz seyi okuyabiliyor ve anlayabiliyorum; fakat orijinale ne kadar bagli kalmak istedigimizle ilgili bir soru isaretine sebep oluyor okuduklarim. 

> Version control is a system that records changes

ile 

> Versiyon kontrol sistemi; dosya ve dosya grupları

cumleleri ayni anlami vermiyor. Versiyon kontrol ile ilgili bir baslikta once versiyon kontrolun ne oldugundan, sonra bunu yapan sistemlerden sonrasinda Git'in bu denklemdeki yerinden bahsediliyor. Benim okudugum ise dogrudan versiyon kontrol sistemine giris ve devami.

Reply:
Merhaba, inceleme için teşekkür ederim.

orjinale bağlılık ve versiyon kontrol / sistemleri sorunsalı doğrultusunda;

> Versiyon kontrol, belirli versiyonların daha sonra çağrılabilmesi için zaman içerisinde bir dosya veya dosya grubundaki değişiklikleri kaydeden bir sistemdir.